src/main/kotlin/guru/stefma/androidartifacts/plugin/AndroidArtifactsPlugin.kt

```diff
@@ -72,25 +70,7 @@ class AndroidArtifactsPlugin : Plugin<Project> {
 
             it.setupMetadata(this, extension)
 
-            it.pom {
-                it.packaging = "aar"
```

Comment:
You have removed the packaing.
We have to keep that.
Otherwise gradle want to try to download a *jar file. When not added as `dependency.name@aar`... Which is a bad behavior...

Reply:
Added again, but on a different line